region/crateria/central.json

```diff
@@ -859,7 +895,7 @@
           },
           "locks": [
             {
-              "unlock": [ "Kill All Enemies" ]
+              "unlock": [ "ZebesAwake", "Kill All Enemies" ]
```

Comment:
`ZebesAwake` goes in this `unlock`'s matching `lock`. Same for the other door.

Reply:
The left door is locked both before ZebesAwake and after when you need to Kill all enemies to open it. The right door only locks after ZebesAwake.

Reply:
You're right. That makes `ZebesAwake` a bit redundant since you can't kill the enemies before they're there, but it doesn't matter.